src/SFA.DAS.AssessorService.Web/Controllers/CertificateVersionController.cs

```diff
@@ -132,8 +133,19 @@ private async Task<IActionResult> SaveViewModel(CertificateVersionViewModel vm,
             var standardVersionChanged = vm.StandardUId != certificate.StandardUId;
             var standardVersion = await _standardVersionClient.GetStandardVersionByStandardUId(vm.StandardUId);
 
+            // Retrieve what versions the EPAO is able to assess
```

Comment:
Unlike most other controllers the set of Certificate controllers does use FluentValidation - so this should use it too - to maintain that 'consistency'

Reply:
There are issues with the existing code to moving this validation into a 'FluentValidation' rule, the session state and the claim being two of them - therefore the fluent validation will not be used in this case and a tech debt will be raised instead.

Reply:
Tech debt ticket added to Standard Versions tech debt epic: https://skillsfundingagency.atlassian.net/browse/SV-603